### PR TITLE
Promisify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
 
-	applescript.execString('quit app "' + app + '"', function (err) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
-		cb();
-	});
+	return pify(applescript.execString)(`quit app "${app}"`);
```
Comment (gillstrom) [inline index.js]: What about using [`run-applescript`](https://github.com/sindresorhus/run-applescript) instead and we can get rid of `pify`?

Comment (kevva) [inline index.js]: Yup, done.

Comment (kevva) [inline index.js]: Fixed!

**Comment by gillstrom** [comment]
Thank you yo!

**Comment by kevva** [comment]
<3

**Pull request merged**